packages/apps/composer-app/src/plugins.tsx

```diff
@@ -274,12 +275,12 @@ export const plugins = ({
 
       const readme = create(DocumentType, {
         name: INITIAL_DOC_TITLE,
-        content: create(TextType, { content: INITIAL_CONTENT.join('\n\n') }),
+        content: makeRef(create(TextType, { content: INITIAL_CONTENT.join('\n\n') })),
```

Comment:
not a blocker but I think in the end it would be nice if object assignments were automatically wrapped in `makeRef` internally to avoid having to do this. afaik assigning an object to a field in another object wouldn't ever make sense as anything other than a ref so I don't think it would be ambiguous